site/cypress/e2e/Cypress-Gradeable/late_submission_warning_messages.spec.js

```diff
@@ -29,9 +29,20 @@ const getRandomGradeableName = () => {
     }
     return randomString;
 };
+const makeNonTeamGradeable = (gradeableName) => {
+    cy.login('instructor');
+    cy.visit(['sample', 'gradeable']);
+    // Enter gradeable info
+    cy.get('#g_title').type(gradeableName);
+    cy.get('#g_id').type(gradeableName);
+    cy.get('#radio_ef_student_upload').check().click();
```

Comment:
This is counter intuitive, as .check acts the same as .click but is used for checkboxes.

Reply:
Only using .check or .click does not work for me. You need to use check with force:true in order to check to work. It is up to you to see which one is better.